package.json

```diff
@@ -36,23 +36,23 @@
     "blacklist"
   ],
   "jest": {
-    "setupTestFrameworkScriptFile": "<rootDir>/jest-setup.js",
+    "setupFilesAfterEnv": ["<rootDir>/jest-setup.js"],
```

Comment:
`setupTestFrameworkScriptFile` is deprecated in favour of `setupFilesAfterEnv`which supports multiple paths.

See the [docs](https://jestjs.io/docs/en/configuration#setupfilesafterenv-array).

Reply:
👍